widgets/src/lib/interfaces/Types.ts

```diff
@@ -1,67 +1,60 @@
 
+/// Coarse-grained task classification.
+///
+/// This type is used to determine which inference API & widget
+/// we want to display for each given model.
+///
+/// As such, they describe the "shape" of each model's API (inputs and outputs)
+/// so the number of different types is not expected to grow very significantly over time.
+///
 /// In each category, order by decreasing specificity
 /// The order can influence which default pipeline tag is affected to a model (if unspecified in model card)
 export enum PipelineType {
 	/// nlp
-	"text-classification" = "text-classification",
-	"token-classification" = "token-classification",
-	"table-question-answering" = "table-question-answering",
-	"question-answering" = "question-answering",
-	"zero-shot-classification" = "zero-shot-classification",
-	"translation" = "translation",
-	"summarization" = "summarization",
-	"conversational" = "conversational",
-	"feature-extraction" = "feature-extraction",
-	"text-generation" = "text-generation",
-	"text2text-generation" = "text2text-generation",
-	"fill-mask" = "fill-mask",
-	"sentence-similarity" = "sentence-similarity",
+	"text-classification"                                     = "Text Classification",
```

Comment:
I think this will break moon-landing code like `moon-landing/server/views/lib/modelInputSnippets.ts` or `server/views/components/InferenceModal/inferenceSnippets.ts` that uses values "as keys". But putting pretty names instead of `key: key` is a good refactoring though

Reply:
yes there will be an associated fix in moon-landing for sue